src/main/resources/graphql/entrySchema.graphqls

```diff
@@ -6,7 +6,7 @@ type Word {
 type Poem {
     id: ID!,
     content: String!,
-    spentTime: Double!,
+    spentTime: Float!,
     word: Word!
 }
 
```

Comment:
@97e57e 
I assume `timeSpent` would be more natural English than `spentTime`.
Are there any lots of side effects from changing the name? If not, I think it is better to rename it to `timeSpent` 😀

Reply:
i will change word, thank you